frontend/src/pages/projects/projectSharing/utils.ts

```diff
@@ -27,3 +27,6 @@ export const firstSubject = (roleBinding: RoleBindingKind): string =>
 
 export const roleLabel = (value: ProjectSharingRoleType): string =>
   value === ProjectSharingRoleType.ADMIN ? 'Admin' : 'Edit';
+
+export const isBiasMetricsEnabled = (dashboardConfig: DashboardConfig) =>
```

Comment:
This function isn't necessary, we already have the `useBiasMetricsEnabled` hook[1] please could you update the PR to use this?

[1] https://github.com/alexcreasy/odh-dashboard/blob/e2f799d38ee6b18c2d224722a6a6fd875a1dd6ca/frontend/src/concepts/explainability/useBiasMetricsEnabled.ts